OWNERS_ALIASES

```diff
@@ -11,3 +11,43 @@ aliases:
   - jottofar
   - vrutkovs
   - wking
+  team-leads:
+  - JAORMX # Infrastructure Security and Compliance
+  - JoelSpeed # Cluster Infrastructure
+  - LalatenduMohanty # Over the Air Updates
+  - Miciah # Network Edge
+  - adambkaplan # Build + Jenkins
+  - ashishkamra # Performance and Latency Sensitive Application Platform
+  - beekhof # Machine Lifecycle
+  - brenton # Technical Release Team
+  - bthurber # Partner Accelerators
+  - celebdor # Ecosystem Edge
+  - dmage # Image Registry
+  - hexfusion # etcd
+  - jaypoulz # Multi Arch
+  - jcantrill # Cluster Logging
+  - jmrodri # Operator SDK
+  - joelddiaz # Hive
+  - jparrill # Portfolio Integration
+  - jsafrane # Storage
+  - jwmatthews # App Migration
+  - kevinrizza # Operator Lifecycle Manager
+  - mandre # Shift on Stack
+  - miabbott # CoreOS
+  - mrmisiex # Kuryr Networking
+  - pedjak # App Services
+  - romfreiman # Single Node OpenShift
+  - rphillips # Node
+  - s-urbaniak # Authentication
+  - sadasu # Metal Platform
+  - selansen # Windows Containers
+  - simonpasquier # Monitoring - In Cluster
+  - sinnykumari # MCO
+  - soltysh # Workloads
+  - sosiouxme # Automated Release Tooling
+  - staebler # OpenShift Installer
+  - stevekuznetsov # Test Platform
```

Comment:
this should be petr muller

Reply:
Yeah, I told Petr that the li config is stale.

Reply:
Backing data was updated, I've refreshed here with fc34a9be48 -> 36b4e72d39, and this entry now correctly references Petr.